resolver-dns/src/main/java/io/netty/resolver/dns/DnsNameResolver.java

```diff
@@ -308,6 +311,22 @@ public void operationComplete(ChannelFuture future) throws Exception {
         });
     }
 
+    private static ResolvedAddressTypes resolvedAddressTypes(InternetProtocolFamily[] internetProtocolFamilies) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Consider using varargs for methods or constructors which take an array the last parameter. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=pmd%3AUseVarargs)